manifests/install.pp

```diff
@@ -5,8 +5,6 @@
   if $nomad::data_dir {
     file { $nomad::data_dir:
       ensure => 'directory',
-      owner  => $nomad::user,
-      group  => $nomad::group,
```

Comment:
Is this really a good idea? Shouldn't it be left in?

Reply:
the user is never used, even the upstream package runs the service as root

Reply:
In that case I would set it to `root` so it is managed.